gradle.html

```diff
@@ -69,5 +69,5 @@ <h2>Run HelloWorld using Gradle</h2>
 
 <div class="alert alert-warning">
     <strong>Note: </strong>
-    Make sure to use gradle version 4.10+ and set the JAVA_HOME environment variable to JDK <span class="JDK_MAJOR">11</span>.
+    Make sure to use gradle version 4.10+ and set the JAVA_HOME environment variable to correct JDK location.
```

Comment:
Probably we need a higher gradle version now... at least 6?

Reply:
We need a Gradle -> Java support table :)

Reply:
Anyway, it doesn't make sense keeping a very old gradle version. JDK 13 requires gradle 5? and 14 6.3.

Reply:
JDK 13 requires gradle 6.0 or later. JDK 14 will require gradle 6.3.

Reply:
I have added a table to show our recommended minimum Gradle version against each JDK.